src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-page.component.html

```diff
@@ -52,66 +52,12 @@ <h1>
-                <option [value]="InstructorSessionResultViewType.GRQ">Group by - Giver &#8594; Recipient &#8594; Question</option>
-                <option [value]="InstructorSessionResultViewType.RGQ">Group by - Recipient &#8594; Giver &#8594; Question</option>
-                <option [value]="InstructorSessionResultViewType.GQR">Group by - Giver &#8594; Question &#8594; Recipient</option>
-                <option [value]="InstructorSessionResultViewType.RQG">Group by - Recipient &#8594; Question &#8594; Giver</option>
-              </select>
-            </div>
-          </div>
-          <div class="report-config-item">
-            <label class="report-config-label control-label mt-md-1">
-              <b ngbTooltip="View results by sections" class="ngb-tooltip-class">Section:</b>
-            </label>
-            <div>
-              <select id="section-dropdown" class="form-control" [(ngModel)]="section">
-                <option value="">All</option>
-                <option *ngFor="let section of sectionsModel | keyvalue" [value]="section.key">
-                  {{ section.key === 'None' ? 'No specific section' : section.key }}
-                </option>
-              </select>
-            </div>
-          </div>
-          <div *ngIf="section" class="report-config-item">
-            <label class="report-config-label control-label mt-md-1">
-              <b ngbTooltip="Choose the way responses for the selected section is viewed" class="ngb-tooltip-class">If showing only a section:</b>
-            </label>
-            <div>
-              <select id="section-type-dropdown" class="form-control" [(ngModel)]="sectionType">
-                <option *ngFor="let sectionType of InstructorSessionResultSectionType | enumToArray" [ngValue]="sectionType">
-                  {{ sectionType | sectionTypeDescription }}
-                </option>
-              </select>
-            </div>
-          </div>
-        </div>
-        <div class="col-md-6" style="margin-top: 10px;">
-          <label class="control-label">
-            <b>Additional settings:</b>
-          </label>
-          <div>
-            <label class="form-check-label" ><input id="include-team-grouping" type="checkbox" [(ngModel)]="groupByTeam" [disabled]="viewType === 'QUESTION'"> Group by Teams</label>
-          </div>
-          <div>
-            <label class="form-check-label" ><input id="include-statistics" type="checkbox" [(ngModel)]="showStatistics" [disabled]="viewType === 'GRQ' || viewType === 'RGQ'"> Show Statistics</label>
-          </div>
-          <div>
-            <label class="form-check-label" ><input id="include-missing-responses" type="checkbox" [(ngModel)]="indicateMissingResponses"> Indicate Missing Responses</label>
-          </div>
-        </div>
-      </div>
-    </div>
+    <tm-view-results-panel [session]="session" [(viewTooltipText)]="viewTooltipText" [(viewType)]="viewType"
```

Comment:
There shouldn't be a need to pass `viewTooltipText` from the page component. Unlike other variables passed here, the text is only relevant within the context of the panel.

Reply:
fixed in f4952ca